sdks/python/apache_beam/testing/test_utils.py

```diff
@@ -144,20 +144,19 @@ def wait_for_topics_created(topics, timeout=60):
 
 
 def _wait_until_all_exist(components, timeout):
-  needs_wait = set(components)
+  unchecked_components = set(components)
   start_time = time.time()
   while time.time() - start_time <= timeout:
-    for c in components:
-      if c in needs_wait and c.exists():
-        needs_wait.remove(c)
-    if len(needs_wait) == 0:
+    exist = set([c for c in unchecked_components if c.exists()])
```

Comment:
How about, writing the opposite way for making it easier to read

```
 unchecked_components = set([c for c in unchecked_components if not c.exists()])
 if len(unchecked_components) == 0:
...
```

Reply:
good suggestion! done.